static/package-lock.json

```diff
@@ -1,8 +1,9479 @@
 {
   "name": "answers-hitchhiker-theme",
   "version": "1.26.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think we don't want to change the lockfile ver until we're sure the CBD containers can handle it